.editorconfig

```diff
@@ -9,7 +9,10 @@ indent_size = 4
 indent_style = space
 end_of_line = lf
 insert_final_newline = true
-trim_trailing_whitespace = true
+trim_trailing_whitespace = false
```

Comment:
Why these changes?

Reply:
Are the newlines at the end of the Blade file that bad? All Blade files have that anyway to keep Git happy.

Reply:
Some of the blade files in this package render an `img` tag. I think it's best if they don't have that extra newline so the in the resulting html page there won't be extra empty lines added.

Reply:
In that case it would make more sense to add specific exceptions for the files where it matters in `.editorconfig` instead of having a global configuration that can introduce issues along the road.

Reply:
I think it's fine for now. This behaviour is wanted for all currently present blade files, and I don't see us adding any new ones soon.